docker/docker-compose.yaml

```diff
@@ -29,6 +29,10 @@ services:
       - redis
       - celeryworker
       - celerybeat
+    volumes:
+      - source: tmp
+        target: /tmp
```

Comment:
Can you pick a more specific place, maybe in /var? /tmp might get filled with system files unrelated to the portal